src/keys.c

```diff
@@ -306,7 +306,13 @@ int SetCipherSpecs(WOLFSSL* ssl)
         ssl->specs.static_ecdh           = 0;
         ssl->specs.key_size              = DES3_KEY_SIZE;
         ssl->specs.block_size            = DES_BLOCK_SIZE;
+/* DES_IV_SIZE is incorrectly 16 in FIPS v2. It should be 8, same as the
```

Comment:
We could just change all these occurrences of `DES_IV_SIZE` to `DES_BLOCK_SIZE`, but I suspect someone may come along in the future, not know about the FIPS v2 gotcha, and change it back to `DES_IV_SIZE`. Therefore, I think it's better to stick with the ifdef approach and a comment explaining.

Reply:
Agreed. And that FIPS des3.h header is immutable.